doc/src/index.md

```diff
@@ -1,6 +1,23 @@
-# Julia 1.0 Documentation
+```@eval
+io = IOBuffer()
+release = isempty(VERSION.prerelease)
+v = "$(VERSION.major).$(VERSION.minor)"
+!release && (v = v*"-$(first(VERSION.prerelease))")
+print(io, """
+    # Julia $(v) Documentation
 
-Welcome to the documentation for Julia 1.0.
+    Welcome to the documentation for Julia $(v).
+
+    """)
+if !release
+    print(io,"""
+        !!! warning "Work in progress!"
+            This documentation is for an unreleased, in-development, version of Julia.
```

Comment:
The comma after "in-development" feels a little weird but I'm not whether it's actually right or wrong grammatically. Doesn't really matter.